Charon/filetypes/GCodeFile.py

```diff
@@ -195,7 +200,25 @@ def __validateGriffinHeader(metadata: Dict[str, Any]) -> None:
             raise InvalidHeaderException("PRINT.SIZE.MIN.[x,y,z] must be set. Ensure all three are defined.")
         if not GCodeFile.__isAvailable(metadata, ["print", "size", "max", ["x", "y", "z"]]):
             raise InvalidHeaderException("PRINT.SIZE.MAX.[x,y,z] must be set. Ensure all three are defined.")
-        
+               
+        # Validate extruder train, part I
+        for index in range(0, 15):
```

Comment:
Why the increment from the original `10` to `15`? And could you perhaps make this a named constant while you're at it?

Reply:
What the user sees in the front-end is a max of 8 extruders. In the engine (and some of the frontend code as well) we assume max 16 extruders in code. I'm not sure where '10' came from.

Yes, a constant would be good.

Reply:
Thanks, that's clear! A named constant with a small comment explaining that bit would indeed be helpful for the future. Thanks 🙏🏻